### ISSUE TITLE
Why i take this certification, mean what is the benefit?

### ISSUE THREAD

**Issue by ghulamMustafaRaza** [nodejs-certified-developer/certification#26]

**Comment by vvscode** [comment]
Not sure if answer this question become actual closest time - I have been waiting for the certification during last 2 years. Each time they say - it's close. 

So first it worth to see certification itself, and only after that it will be possible to say if it worth to pass it.

**Comment by cbmarc** [comment]
Yeah I think that the worst thing of all regarding this nodejs certification is actually the **SILENCE**.
